prow/config/plugins.yaml

```diff
@@ -41,3 +41,10 @@ config_updater:
       name: config
     prow/config/plugins.yaml:
       name: plugins
+    prow/config/labels.yaml:
+      name: labels
+      key: labels.yaml
```

Comment:
Why do we need this in the cluster? I haven’t deployed a label sync pod, yet anyway

Reply:
See above about who can run CI. What restricts the `approved` and `lgtm` labels from being applied? I can't figure it out  from the prow docs, or looking at the label plugin code.

Reply:
Approved and lgtm are handled by separate plugins.

Reply:
Maybe it's not a good idea then to enable the label via prow

Reply:
Yeah - I just commented the same in another thread.  AFAICT, there's no access control around "/label".  Sorry to send you on a goose chase, but I guess we should leave this one as something only manually applied.

https://github.com/kubernetes/test-infra/blob/master/prow/plugins/label/label.go

Reply:
> goose chase

Another good reason to update and get /honk!

Reply:
It's no goose, but /woof